tests/testpkgs/savanna.nix

```diff
@@ -1,5 +1,9 @@
-{ lib, stdenv, fetchurl, gamin }:
-
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gamin,
+}:
```

Comment:
For example here:
- Why does it enforce multiple lines? Also the commas in the end of the line looks weird.
- Where did the extra new line go?